### ISSUE TITLE
Using Disqus beyond Tumblr

### ISSUE THREAD

**Issue by billbennettnz** [snarfed/bridgy#639]
If Bridgy can pull webmentions from Tumblr via Disqus, that implies it shouldn't be hard to pull all webmentions from Disqus. How hard would that be to add as a feature?

At present posts on my blog are reposted on other sites. One of these sites uses Disqus for comments. It's long been my goal to consolidate both sets of comments. That's a longer term project, but adding general Disqus webmentions to Bridgy would be a good start.

**Comment by snarfed** [comment]
interesting! bridgy doesn't actually pull webmentions from tumblr/disqus right now; it _pushes to_ (ie writes to) disqus.

to clarify your feature request @billbennettnz, you'd like bridgy to detect your blog posts that are syndicated (reposted) across sites and synchronize the disqus comments across them?

definitely an interesting feature request. probably not something i'd personally work on anytime soon, but i'll happily merge PRs if you're interested in contributing! [details here.](https://github.com/snarfed/bridgy/#development) i don't even know if disqus has a read API, but that'd be the first thing to check.

**Comment by billbennettnz** [comment]
I'm no developer (the last code I wrote was in Forth or Pascal in the 1980s) but I'm code-literate enough to dig around the Disqus API and see what can be picked-up.

**Comment by billbennettnz** [comment]
Yes. It looks like you can pull data from Disqus with a GET command. https://disqus.com/api/docs/requests/
